src/test/java/io/thinkit/edc/client/connector/ApplicationObservabilityTest.java

```diff
@@ -22,10 +25,7 @@ void setUp() {
         applicationObservability = client.applicationObservability();
     }
 
-    @Test
-    void should_check_health() {
-        var result = applicationObservability.checkHealth();
-
+    void health_status_response(Result<HealthStatus> result) {
```

Comment:
please use camelCase naming format for these methods, mark them as `private` and put them at the bottom of the class (the tests should be visible first)